test/integration/coronavirus_local_restrictions_test.rb

```diff
@@ -362,134 +92,17 @@ def then_i_see_the_results_with_conventional_postcode_formatting
-    assert page.has_text?(@area)
-    assert page.has_text?(I18n.t("coronavirus_local_restrictions.results.level_two.changing_alert_level", area: @area))
-  end
-
-  def then_i_see_the_results_page_for_level_three_with_changing_restriction_levels
-    assert page.has_text?(@area)
-    assert page.has_text?(I18n.t("coronavirus_local_restrictions.results.level_three.changing_alert_level", area: @area))
-  end
-
-  def then_i_see_an_out_of_date_warning
-    assert page.has_text?(I18n.t("coronavirus_local_restrictions.out_of_date_warning"))
-  end
-
-  def then_i_do_not_see_an_out_of_date_warning
-    assert_not page.has_text?(I18n.t("coronavirus_local_restrictions.out_of_date_warning"))
-  end
-
-  def and_there_is_metadata
-    assert page.has_css?("meta[property='og:title'][content='#{I18n.t('coronavirus_local_restrictions.lookup.meta_title')}']", visible: false)
-    assert page.has_css?("meta[name='description'][content='#{I18n.t('coronavirus_local_restrictions.lookup.meta_description')}']", visible: false)
-    assert page.has_css?("link[rel='canonical'][href='http://www.example.com/find-coronavirus-local-restrictions']", visible: false)
-  end
-
-  def stub_local_restriction(postcode:,
-                             name: "Tatooine",
-                             gss: SecureRandom.alphanumeric(10),
-                             country_name: "England",
-                             current_alert_level: nil,
-                             future_alert_level: nil)
-    areas = [
-      {
-        "gss" => gss,
-        "name" => name,
-        "type" => "LBO",
-        "country_name" => country_name,
-      },
-    ]
-    stub_mapit_has_a_postcode_and_areas(postcode, [], areas)
-
-    current_restriction = if current_alert_level
-                            { "alert_level" => current_alert_level,
-                              "start_date" => 1.week.ago.to_date,
-                              "start_time" => "10:00" }
-                          end
-
-    future_restriction = if future_alert_level
-                           { "alert_level" => future_alert_level,
-                             "start_date" => 1.week.from_now.to_date,
-                             "start_time" => "10:00" }
-                         end
-
-    local_restriction = LocalRestriction.new(gss, {
-      "name" => name,
-      "restrictions" => [current_restriction, future_restriction].compact,
-    })
-    LocalRestriction.stubs(:find).with(gss).returns(local_restriction)
-  end
-
-  def stub_no_local_restriction(
-    postcode:,
-    name: "Tatooine",
-    gss: SecureRandom.alphanumeric(10),
-    country_name: "England"
-  )
-    areas = [
-      {
-        "gss" => gss,
-        "name" => name,
-        "type" => "LBO",
-        "country_name" => country_name,
-      },
-    ]
-    stub_mapit_has_a_postcode_and_areas(postcode, [], areas)
-
-    LocalRestriction.stubs(:find).with(gss).returns(nil)
-  end
```

Comment:
This is great - it makes the PR a net code reduction of 200 lines and makes this file a much easier to digest ~100 lines from ~500.